### ISSUE TITLE
Duplicated Object Property Definition

### ISSUE THREAD

**Issue by kdaveed** [RDFBones/RDFBones-O#93]
The following object properties from OBI-RDFBonesSubset.owl : 

<owl:ObjectProperty rdf:about="http://purl.obolibrary.org/obo/BFO_0000050"/>"
<owl:ObjectProperty rdf:about="http://purl.obolibrary.org/obo/BFO_0000051"/>
<owl:ObjectProperty rdf:about="http://purl.obolibrary.org/obo/IAO_0000136"/>
<owl:ObjectProperty rdf:about="http://purl.obolibrary.org/obo/OBI_0000293"/>
<owl:ObjectProperty rdf:about="http://purl.obolibrary.org/obo/OBI_0000299"/>
<owl:ObjectProperty rdf:about="http://purl.obolibrary.org/obo/OBI_0001938"/>
<owl:ObjectProperty rdf:about="http://purl.obolibrary.org/obo/RO_0000053"/>

appear as well in the RDFBones.owl. The only difference is that in the RDFBones.owl the entries do not start with "<owl:ObjectProperty" string but with "<ObjectProperty". Is there any difference between the two notations?

**Comment by cuboideum** [comment]
Elements from other ontologies should be declared as what they are (proerties, classes, etc.). Otherwise the reasoner would not know how to handle them, as URIs as such do not contain this information.

However, the syntax differences observed by @kdaveed are strange, indeed. I do not know if both notations are valid. I only checked my local file after having edited it with Protégé. There, I found all declarations spelled correctly as owl:ObjectProperty. The only explanation I could imagine would be that Protégé is inconsistent in how it saves files. At the same occasion I also found out that Protégé apparently transfers annotation axioms from one object property to another when they are marked as inverse properties.

As a consequence, I will stop using Protégé as editor and only as a browser.
